pkg/verifier/aws/aws_verifier.go

```diff
@@ -521,16 +522,35 @@ func (a *AwsVerifier) CreateSecurityGroup(ctx context.Context, tags map[string]s
 // ipPermissionFromURL generates an EC2 IpPermission (for use in sec. group rules) from a given IP address
 // -based http(s) URL (e.g. "https://10.0.8.1:8080") with the given human-readable description (ipPermDescription)
 func ipPermissionFromURL(ipUrlStr string, ipPermDescription string) (*ec2Types.IpPermission, error) {
-	// First validate URL by parsing it
+
+	//validate URL by parsing it
 	ipUrl, err := url.Parse(ipUrlStr)
 	if err != nil {
 		return nil, err
 	}
 
-	// Then attempt to extract IP address
-	ipAddr, err := netip.ParseAddr(ipUrl.Hostname())
+	//extract the hostName from the URL
+	hostValue := ipUrl.Hostname()
+	ipFromDomain := hostValue
```

Comment:
nitpick - you could get away with only using one variable here if you wanted by doing `ipFromDomain := ipUrl.Hostname()` here, and then passing it into the `validate.Var` call instead of `hostValue`.

Reply:
ohh you right. testing.